frontend/projects/valtimo/case/src/lib/case-inspection/tabs/logs/logs-tab.component.scss

```
@@ -0,0 +1,192 @@
+      font-size: var(--cds-helper-text-01-font-size);
+      word-break: break-all;
+    }
+  }
+
+  &__detail-stacktrace {
+    margin: 0;
+    padding: var(--cds-spacing-03);
+    background-color: var(--cds-background);
+    border: 1px solid var(--cds-border-subtle);
+    white-space: pre-wrap;
+    word-break: break-word;
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)